src/main/java/com/redhat/rhjmc/containerjfr/net/web/handlers/RecordingOptionsPatchHandler.java

```diff
@@ -98,19 +98,17 @@ void handleAuthenticated(RoutingContext ctx) throws Exception {
                         getConnectionDescriptorFromContext(ctx),
                         connection -> {
                             MultiMap attrs = ctx.request().formAttributes();
-                            if (attrs.contains("toDisk")) {
-                                OptionKey.fromOptionName("toDisk")
-                                        .ifPresent(key -> customizer.set(key, attrs.get("toDisk")));
-                            }
-                            if (attrs.contains("maxAge")) {
-                                OptionKey.fromOptionName("maxAge")
-                                        .ifPresent(key -> customizer.set(key, attrs.get("maxAge")));
-                            }
-                            if (attrs.contains("maxSize")) {
-                                OptionKey.fromOptionName("maxSize")
-                                        .ifPresent(
-                                                key -> customizer.set(key, attrs.get("maxSize")));
+                            String[] keys = {"toDisk", "maxAge", "maxSize"};
+                            for (String key : keys) {
```

Comment:
`keys` can be inlined since it isn't used anywhere lower down in the scope. Or, to be extra fancy, you could do:

```
Arrays.asList("toDisk", "maxAge", "maxSize").forEach(key -> {
  if (attrs.contains(key)) {
  ...
  }
...
});
```

Reply:
Ok, that's definitely better!